docs/modules/servers/pages/basic/help.adoc

```diff
@@ -1,5 +1,54 @@
 = James Basic Server &mdash; Where to get help
 :navtitle: Help
 
-(TODO)
+////
 
+This information is based on the email discussion from here:
+  --> https://www.mail-archive.com/server-dev@james.apache.org/msg66420.html
+
+However, I am not satisfied that I have properly captured the community's
+opinion, and there are only a few people who have contributed to the
+thread.
+
+Would be good to have some comments on this content.
+
+It could just be that I don't like the content. But if that is what the
+community thinks, then that is what we should write.
+
+////
+
+== Warning
+
+Please note that if you want to try to use James, you are pretty
+much on your own.
+
+James is intended for use only by the current committers, or perhaps only by
+competent developers only who have the ability to understand the code as it
+exists now and can make changes themselves without any help. What you see
+is what you get. Take it or leave it.
+
+The James Community is made up of people who "scratch their own itch".
+If you have a problem with the code or the documentation, then please
+step up and fix it rather than asking for help.
+
+If you are a user who wants to try to use James, you should seriously
+consider hiring a professional organization to help you. Most likely,
+James is not for you.
+
+You can try filing issues, but there is absolutely no guarantee it will be
+fixed. If you are lucky and a developer has the same problem as you, then
+it may be fixed. Or if one of the developers is feeling particularly generous and wants
+to volunteer their valuable time to help you, it may happen. There is no 
+way to know unless you try.
+
+On the other hand, the developers would **love** for you to do stuff that
+they are not interested in doing, like documentation. If you don't know anything
```

Comment:
`the developers would **love** for you to do stuff that they are not interested in doing, like documentation.`

That is not the right way to put it.

Documentation is an easy onboarding step. The target audience have a better understanding of the structure and information desired than the developers, hence user help is much wanted.

Reply:
Same as above, this was the result of one of our discussions. I also do not like this content.

I think the community should have another discussion regarding it's objectives. I think the community should be more aware of what it is willing / unwilling to do, and I think it should be documented honestly. It should also be presented in a way that it is useful for the reader and addresses their concerns.

How would you propose we go about redoing this discussion?

Reply:
Having spent my day doing documentation, and having committing several weeks lunch pauses doing so, "documentation" is something "that I am interested in doing".

We, as a community, agreed to be reworking documentation (cf conf call notes).

We are devoting time and efforts to it.

That don't change the fact that documentation is nice for onboarding.

Reply:
I agree with these comments. I have started a discussion on the list in the hope of reaching a consensus. I will resolve this comment for now.